nautobot/ipam/models.py

```diff
@@ -613,21 +624,25 @@ def get_first_available_ip(self):
         return "{}/{}".format(next(available_ips.__iter__()), self.prefix.prefixlen)
 
     def get_utilization(self):
-        """
-        Determine the utilization of the prefix and return it as a percentage. For Prefixes with a status of
-        "container", calculate utilization based on child prefixes. For all others, count child IP addresses.
+        """Get the child prefix size and parent size.
+
+        For Prefixes with a status of "container", get the number child prefixes. For all others, count child IP addresses.
+
+        Returns:
+            UtilizationData (namedtuple): (numerator, denominator)
```

Comment:
Good to see this, but let's make please this return documentation be uniform across all the places it is in docstrings.

Reply:
Agreed! Will get it updated today.